Common/Data/Custom/Quiver/QuiverWikipedia.cs

```diff
@@ -0,0 +1,158 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Util;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    /// <summary>
+    /// Twitter follower counts for the specified company
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class QuiverWikipedia : BaseData
+    {
+
+        /// <summary>
+        /// The date of the follower count
+        /// </summary>
+        [ProtoMember(10)]
+        [JsonProperty(PropertyName = "Date")]
+        [JsonConverter(typeof(DateTimeJsonConverter), "yyyy-MM-dd")]
+        public DateTime Date { get; set; }
+
+
+
+        /// <summary>
+        /// The follower count on the given date
+        /// </summary>
+        [ProtoMember(11)]
+        [JsonProperty(PropertyName = "Views")]
+        public decimal? PageViews { get; set; }
+
+        /// <summary>
+        /// The follower count % change over the week prior to the date
+        /// </summary>
+        [ProtoMember(12)]
+        [JsonProperty(PropertyName = "pct_change_week")]
+        public decimal? WeekPercentChange { get; set; }
+
+
+        /// <summary>
+        /// The follower count % change over the month prior to the date
```

Comment:
We might want to add a note that we're providing percentage values as whole numbers, i.e. 100% == 1.0